extensions-core/druid-ranger-security/pom.xml

```
@@ -160,6 +160,12 @@
                     <groupId>org.elasticsearch.plugin</groupId>
                     <artifactId>*</artifactId>
                 </exclusion>
+                <!-- jackson-jaxrs is used by ranger-plugins accessing
+                RangerRESTClient class. This should not be needed in an authorizer -->
+                <exclusion>
+                    <groupId>org.codehus.jackson</groupId>
```

I believe there is a typo here

```suggestion
                    <groupId>org.codehaus.jackson</groupId>
```